src/pages/home/report/ReportActionCompose.js

```diff
@@ -301,6 +301,7 @@ class ReportActionCompose extends React.Component {
     }
 
     onSelectionChange(e) {
+        LayoutAnimation.configureNext(LayoutAnimation.create(50, LayoutAnimation.Types.easeInEaseOut, LayoutAnimation.Properties.opacity));
```

Comment:
@allroundexperts I guess it's safe to remove this line? Is it doing anything useful?

Reply:
I assume https://github.com/Expensify/App/pull/19933 is already the equivalent here, so we can move ahead and remove it?